### DISCUSSION TITLE
Sandboxie Plus migration guide on the sandboxie-docs repository

### DISCUSSION THREAD

**Discussion by isaak654** [sandboxie-plus/Sandboxie#1969] [Various ideas and suggestions] [answered]
@DavidXanatos 
Regarding the [Sandboxie Plus migration guide](https://sandboxie-plus.com/sandboxie-plus-migration-guide/), I think it would be reasonable to create a replica on the [sandboxie-docs](https://github.com/sandboxie-plus/sandboxie-docs) repository.

See also https://github.com/sandboxie-plus/Sandboxie/discussions/1756

**Comment by DavidXanatos** [comment, marked as answer]
done: https://github.com/sandboxie-plus/sandboxie-docs/blob/main/Content/PlusMigrationGuide.md
